packages/odyssey-react/src/components/Status/index.tsx

```diff
@@ -53,18 +53,12 @@ const Status: FunctionComponent<Props> = (props) => {
     variant = "neutral",
   } = props;
 
-  const componentClass = useCx(
-    "ods-status",
-    `is-ods-status-${variant}`,
-    { "is-ods-status-label-hidden": labelHidden }
-  );
-
   return (
-    <div className={componentClass} role="status">
-      <span className="ods-status--label">
+    <div className={styles.status} role="status">
+      <span className={labelHidden ? styles['label-visually-hidden'] : styles.label}>
```

Comment:
does this pass linting? I would have expected the classname to be `labelVisuallyHidden` to avoid the need for bracket notation lookup here

Reply:
No, but I didn't have inline linting turned on for css/sass so I didn't catch it until the build failed. All set now